18/README.md

```diff
@@ -0,0 +1,119 @@
+```
+shortname: 18/tm_stress
+Name: Tendermint Performance Profiling
+type: Informational
+status: Raw
+Editors: Zachary Bowen <zach@bigchaindb.com>
+Contributors: 
+```
+
+# Tendermint Performance Profiling
+
+## Abstract
+This document contains the results of an in-depth benchmarking study of the Tendermint consensus layer. We use only Tendermint native tooling, and attempt to independently replicate their claim of [10500 txs/s][tm_benchmark_claim].
+
+## Motivation
+BigChainDB has quoted the Tendermint performance claim in our own documentation, and to date we have not independently verified this figure. As this represents a potential bottleneck for our own performance, and given that we have now released BigchainDB Beta, it is critical that we get a better understanding of the performance capabilities of Tendermint.
+
+This investigation is further motivated as a natural out-growth of [BEP-10], which makes the assumption that tighter integration with Tendermint will result in performance improvements for BigchainDB. This assumption only holds if Tendermint is *not* the performance bottleneck for our system. Given the complexity of the consensus problem, it is reasonable to test this assumption before planning any engineering projects around it.
+
+## Test System
+All tests were run on hardware with the following specifications
+
+* Processor: Intel(R) Xeon(R) CPU E5-2650 v3 @ 2.30GHz (10 cores)
+* Memory: 15GB
+* OS: Ubuntu 18.04 LTS
+* Tendermint version: 0.20.0-27bd1dea
+
+## Tests
```

Comment:
Did you test with `create_empty_blocks=false`?

May be also worth trying to run with `recheck=false`. A blind guess - a rate would go up higher but the general degradation pattern will stay.

Reply:
I tried it with `create_empty_blocks=false` and `true`. It didn't make a difference, since while the test is running, there is a constant stream of requests, and hence no empty blocks get minted.

Reply:
Added a paragraph to make this explicit.

Reply:
Great, thanks!